layouts/shortcodes/shared/projects-card-deck.html

```diff
@@ -0,0 +1,24 @@
+<div class="card-deck">
+
+    {{ range $.Site.Data.projects }}
+    {{ partial "project-card.html" . }}
+    {{ end }}
+
+    <div class="card">
```

Comment:
Could this card also be a "project" from data? Or at least use the same partial?

Reply:
It shouldn't be part of the project data (because then everywhere that wanted to use the project data would have to check to see if this "dummy" project should be included), but the data can be passed to the partial. I've done that.